Purchases/RCCrossPlatformSupport.h

```diff
@@ -6,19 +6,31 @@
 //  Copyright © 2019 RevenueCat. All rights reserved.
 //
 
-#if TARGET_OS_IPHONE
-#import <UIKit/UIKit.h>
+#if TARGET_OS_IOS || TARGET_OS_TV
 #define APP_DID_BECOME_ACTIVE_NOTIFICATION_NAME UIApplicationDidBecomeActiveNotification
 #define APP_WILL_RESIGN_ACTIVE_NOTIFICATION_NAME UIApplicationWillResignActiveNotification
-#elif TARGET_OS_MAC
-#import <AppKit/AppKit.h>
+#elif TARGET_OS_OSX
 #define APP_DID_BECOME_ACTIVE_NOTIFICATION_NAME NSApplicationDidBecomeActiveNotification
 #define APP_WILL_RESIGN_ACTIVE_NOTIFICATION_NAME NSApplicationWillResignActiveNotification
+#elif TARGET_OS_WATCH
+#define APP_DID_BECOME_ACTIVE_NOTIFICATION_NAME NSExtensionHostDidBecomeActiveNotification
+#define APP_WILL_RESIGN_ACTIVE_NOTIFICATION_NAME NSExtensionHostWillResignActiveNotification
 #endif
+
+#if TARGET_OS_IPHONE
+#import <UIKit/UIKit.h>
+#elif TARGET_OS_OSX
+#import <AppKit/AppKit.h>
+#endif
+
 #if TARGET_OS_MACCATALYST
 #define PLATFORM_HEADER @"uikitformac"
-#elif TARGET_OS_IPHONE
+#elif TARGET_OS_IOS
 #define PLATFORM_HEADER @"iOS"
-#elif TARGET_OS_MAC
+#elif TARGET_OS_OSX
```

Comment:
here's where using `MAC` would be problematic - it'd always be true and we'd never go into the other cases